sdk-core/src/main/java/care/data4life/sdk/RecordService.kt

```diff
@@ -262,20 +292,20 @@ class RecordService(
     }
 
     private fun encryptTagsAndAnnotations(
-            painTags: HashMap<String, String>,
+            plainTags: HashMap<String, String>,
             plainAnnotations: List<String>
```

Comment:
what does plain mean? not encoded?

Reply:
Plain mean in this case not encoded and not encrypted...but you point me here to something else thx!